Rocket-Nozzle-Design-and-Analysis/Rocket_Nozzle_Design_and_Analysis.tex

```diff
@@ -0,0 +1,235 @@
+  There will be sensors placed in strategic locations that will assist in the data analysis to allow the team to understand conditions of the fluid as it passes through the nozzle and exits. The results
+  of the experiment will be summarized in a final report and presented at Imagine RIT, depending on the completion date.
+\end{abstract}
+
+\label{sec:nomenclature}
+\newcommand{\nomunit}[1]{%
+\renewcommand{\nomentryend}{\hspace*{\fill}#1}}
+\renewcommand{\nompreamble}{
+    % If you include mathematical expressions or express variables in the design document, list them with their corresponding definitions here as a list.
+    % The two lines below make it look nice when defining units/values to constants.
+
+    % Note that math terms and non-math terms are separated and alphabetized, regardless of the order in which they are defined. (Recall terms \$like this\$ are in the math environment)
+    % Read more about advanced nomenclature formatting here:\\
+    % \url{https://www.sharelatex.com/learn/Nomenclatures}
+  }
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
+% Below are examples of using nomenclature for math symbols and constants or units
+% \nomenclature{$\dot{m}$}{Mass flow rate
+%   \nomunit{\,\si{\kilo\gram\per\second}}}
+% \nomenclature{$c$}{Speed of light
+%  \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+\section{Introduction}
+\label{sec:introduction}
+\IEEEPARstart{P}{ropulsion} systems are very difficult to design and test. The SPEX propulsion team has worked for two semesters, Spring 2017 & Fall 2017, to design and test a cold gas propulsion system.
+There was moderate success in the design and testing of this system. The successes included, nozzle design, nozzle fabrication, test stand fabrication, and experimentation.
+The nozzles were designed through the Aerospike software given to us by an advisor, Dr. Dorin Patru. The nozzles were fabricated by methods of additive manufacturing.
+The test stand was properly designed and fabricated to work with thrust levels up to 10 Newtons. The experimentation was successful because
+the team was able to test multiple nozzles with different operating characteristics. There developed a problem in the fabrication of conical nozzles that the team designed using the analytical rocket equations.
+The nozzles for the thrust levels were too small. The fabrication process would be inhibitively difficult. The solution to this is an increase in the thrust level.
+ This project will focus on the design of a conical nozzle for a 100 Newton thrust force test.
+
+\section{Primary Objective}\label{sec:primary-obj}
+The primary goal of this project is to use the analytical rocket equations to design a conical rocket nozzle to produce 100 Newtons of thrust.
+The team will design a new stand that is capable of securely fastening a propulsion test of 200 Newtons.
```

Comment:
I can say from experience that a test stand is its own bundle of ~~problems~~ joy. Since this PDD proposes them both, I strongly recommend having the test stand be a secondary objective and discuss its requirements and stuff on its own.